### DISCUSSION TITLE
Tailwind classnames don't show up in DOM for only a specific div

### DISCUSSION THREAD

**Discussion by clearly-outsane** [vercel/next.js#40696] [Help]
I'm not sure how to make this reproduceable or why it's even happening. But this one specific div doesn't get tailwind classes.
![image](https://user-images.githubusercontent.com/44093134/191112144-28a293cf-8103-4fa7-b065-728eb8962347.png)
![image](https://user-images.githubusercontent.com/44093134/191112158-f9091207-566c-4df1-86e2-586bf4142fd8.png)

As you can see in the code that div has classnames, but in the DOM, it has no class. If I edit the classname and save, it shows up on hot reload - but once I refresh the page it disappears again. I'm using TailwindCSS with Nextjs. Everything else on the page renders as intended, it's only this random div.

Could someone direct me as to how I can beginning finding a way to reproduce this? So that if it's a bug, it can be fixed

**Comment by icyJoseph** [comment]
Got a repo to take a look at? At first glance nothing is wrong. I would imagine that something is mutating divs, or some other kind of side-effect. 

Also please post code, screenshots are not very nice. AFAIK, I've correctly copied that exact class name into another project and it resolved it correctly, so there's no parsing problems going on.  

Given the problem I'd say we have to look at the project widely. We might have a plugin that's causing conflicts, some kind of side-effect removing every element in the class list. Perhaps you have a browser extension or something that's interfering? It could be so many things... 

Try it in incognito mode, or make a curl request for your page, or if you use VSCode use Thunder Client or Rest Client to see if the problem also happens there.

Can you remove surrounding code and still see the issue? Can you remove plugins and still the issue?



To say it is a bug you'd need to create a reproduction example or repository.
